includes/drush/custom/diffy.drush.md

```diff
@@ -0,0 +1,96 @@
+# Diffy Drush Commands
+
+These commands are purposed to allow easy Diffy interaction before, during and
+after deployments.
+
+## Requirements
+- Drush 8 >=
+- `export DIFFY_API_KEY=xxxxxxxxxxxxxxxxxxxxxxxxxx`
+
+## Commands
+
+### diffy-refresh-token
+
+This command requires the **DIFFY_API_KEY** environment variable to be set. If
+set correctly this command will save a Diffy token inside of the database under
+the Drupal variable name: **diffy_token**. This token will be used for
+subsequently executed Diffy commands.
+
+
+### diffy-project-snapshot
+
+This command allows you to take a snapshot of the current website IF the
+**diffy_project_id** Drupal variable is defined. If not you need to provide it
```

Comment:
What is the plan to align the projects with this value?

Reply:
I'm not sure. We should have something that's maintainable. But this is a problem without a real inventory.

I could make an extra target that creates the project and sets the project id if no id is provided.

Reply:
Maybe we should start think about create our dinamic Inventory using the GitHub API and topics information.